kubelet-log-config/kubelet-log-config.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kubelet-log-config
+  namespace: default
+  labels:
+    k8s-app: kubelet-log-config
+spec:
+  selector:
+    matchLabels:
+      name: kubelet-log-config
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        name: kubelet-log-config
+    spec:
+      nodeSelector:
+        k8s-app: kubelet-log-config
+      hostPID: true
+      containers:
+        - name: startup-script
+          image: gcr.io/google-containers/startup-script:v1
```

Comment:
this image is quite old. It is great as it has systemctl installed and doesn't need the `nsenter`. But it is not ideal to run such an old image here